build-octotiger.sh

```diff
@@ -7,15 +7,14 @@ set -ex
     ${OCT_WITH_KOKKOS_SCALAR:?} ${CUDA_SM:?} ${OCT_ARCH_FLAGS:?}
 
 
-DIR_SRC=${SOURCE_ROOT}/octotiger
-DIR_BUILD=${INSTALL_ROOT}/octotiger/build
+DIR_SRC=${SOURCE_ROOT}/octotiger-kokkos
+DIR_BUILD=${INSTALL_ROOT}/octotiger/build-kokkos
 #DIR_INSTALL=${INSTALL_ROOT}/octotiger
 
 if [[ ! -d ${DIR_SRC} ]]; then
     git clone https://github.com/STEllAR-GROUP/octotiger.git ${DIR_SRC}
     pushd ${DIR_SRC}
     git checkout master
-    git submodule update --init --recursive
```

Comment:
Why stop checking out the testdata? This would require us to adapt all Jenkins CI pipelines to add this as an additional step (in addition to do this manually for all of our other builds).